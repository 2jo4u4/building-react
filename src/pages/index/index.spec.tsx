import React from 'react'
import App from './index'
import renderer from 'react-test-renderer'

test('Index App', () => {
  const component = renderer.create(<App />)
  const tree = component.toJSON() as renderer.ReactTestRendererJSON
  expect(tree).toMatchSnapshot()
})
