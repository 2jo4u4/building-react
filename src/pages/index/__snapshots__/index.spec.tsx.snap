// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index App 1`] = `
<div>
  Index
  <button
    onClick={[Function]}
  >
    getData
  </button>
</div>
`;
